modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -30,6 +30,38 @@ definition {
 				objectName = "CustomObject187390",
 				pluralLabelName = "Custom Objects 187390");
 
+			Picklist.addPicklistViaAPI(picklistName = "Status");
+
+			for (var statusName : list "open,review,closed") {
+				Picklist.addPicklistItemViaAPI(
+					itemKey = ${statusName},
+					itemName = ${statusName},
+					picklistName = "Status");
+			}
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 187390");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Picklist",
+				fieldPicklist = "Status",
+				fieldType = "Picklist");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Picklist State Manager",
+				fieldPicklist = "Status",
+				fieldType = "Picklist",
+				itemName = "open",
+				markAsState = "true");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Multiselect Picklist",
+				fieldPicklist = "Status",
+				fieldType = "Multiselect Picklist");
+
```

Comment:
Recently the macro `ObjectAdmin.addObjectFieldViaUI` was [changed](https://github.com/brianchandotcom/liferay-portal/pull/142449) with the addition of the step:

```
WaitForVisible(
			key_fieldLabel = ${fieldLabel},
			locator1 = "ObjectAdmin#VIEW_OBJECT_FIELD_LABEL");
```

This causes our test to fail, since before adding fields via the UI, many other fields are added via the API, causing the fields via the UI to be allocated on the last pages and not seen on the first page for the assert.

It is questionable whether this extra step would even be necessary since the [test](https://github.com/brianchandotcom/liferay-portal/pull/142449/commits/28bb673843efc59a0163e85f3e11ebc6c32234b7) added in PR with this modification passes in ci even without this wait (as you can see [here](https://testray.liferay.com/home/-/testray/case_results/2446703846?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2446701950%26testrayRunId%3D2446703820%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)).

To avoid another change in the macro I just changed the order in which the fields are added, making the test add them first via the UI and then via the API.